ResearchKit/Common/ORKTask.h

```diff
@@ -167,6 +167,13 @@ ORK_AVAILABLE_DECL
 - (ORKTaskProgress)progressOfCurrentStep:(ORKStep *)step withResult:(ORKTaskResult *)result;
 
 /**
+ Validates the task parameters.
+ 
+ This method should be used to check that all the task parameters are correct. E.g., `ORKOrderedTask` makes sure that all its step identifiers are unique. It should throw an exception if validation fails.
```

Comment:
Of course the same consideration would apply to `ORKStep`, so we could do that on a follow-on issue.